tests/unit/resizable/core.js

```diff
@@ -246,41 +246,29 @@ QUnit.test( "nested resizable", function( assert ) {
 
 QUnit.test( "Resizable with scrollbars and box-sizing: border-box", function( assert ) {
 	assert.expect( 4 );
-
-	$( "<style> * { box-sizing: border-box; } </style>" ).appendTo( "#qunit-fixture" );
-
-	//Both scrollbars
-	var elementContent = $( "<div>" )
-			.css( {
-				width: "200px",
-				height: "200px",
-				padding: "10px",
-				border: "5px",
-				borderStyle: "solid",
-				margin: "20px"
-			} )
-			.appendTo( "#resizable1" ),
-	element = $( "#resizable1" ).css( "overflow", "auto" ).resizable(),
-	handle = ".ui-resizable-se";
-
-	testHelper.drag( handle, 10, 10 );
-	assert.equal( element.width(), 110, "element width (both scrollbars)" );
-	assert.equal( element.height(), 110, "element height (both scrollbars)" );
-
-	//Single (vertical) scrollbar.
-	elementContent.css( "width", "50px" );
-	testHelper.drag( handle, 10, 10 );
-	assert.equal( element.width(), 120, "element width (only vertical scrollbar)" );
-	assert.equal( element.height(), 120, "element height (only vertical scrollbar)" );
+	testResizableWithBoxSizing( assert, true, false );
 } );
 
 QUnit.test( "Resizable with scrollbars and box-sizing: content-box", function( assert ) {
 	assert.expect( 4 );
+	testResizableWithBoxSizing( assert, false, false );
```

Comment:
See https://ariya.io/2011/08/hall-of-api-shame-boolean-trap; let's use named parameters for readability:
```suggestion
	testResizableWithBoxSizing( assert, {
		isBorderBox: false,
		applyScaleTransform: false
	} );
```
These are tests, we don't need to optimize for size or extreme performance.